randovania/games/dread/json_data/Ghavoran.txt

```diff
@@ -319,14 +353,16 @@ Extra - asset_id: collision_camera_003
   > Pickup (Missile Tank)
       All of the following:
           Morph Ball
-          Flash Shift or Grapple Beam or Speed Booster
+          Flash Shift or Grapple Beam or Space Jump or Speed Booster
   > Tile Group (POWERBEAM) 3
       Morph Ball
 
 > Floating Platform; Heals? False
   * Layers: default
   > Dock to Blue EMMI Introduction
-      Grapple Beam or Spider Magnet or Simple IBJ or Use Spin Boost
+      Any of the following:
+          Spider Magnet or Walljump (Beginner) or Simple IBJ or Use Spin Boost
+          Grapple Beam and Movement (Beginner)
```

Comment:
Same comment here about if we want Movement (Beginner) or not - though that grapple point looks higher and more annoying to reach, so maybe it doesn't need to be consistent with the artaria and dairon examples I listed?